Sources/Conformance/conformance.pb.swift

```diff
@@ -175,15 +175,15 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
       return nil
     }
 
-    fileprivate func traverse(visitor: SwiftProtobuf.Visitor, start: Int, end: Int) throws {
+    fileprivate func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V, start: Int, end: Int) throws {
       switch self {
       case .protobufPayload(let v):
         if start <= 1 && 1 < end {
-          try visitor.visitSingularField(fieldType: SwiftProtobuf.ProtobufBytes.self, value: v, fieldNumber: 1)
+          try visitor.visitSingularBytesField(value: v, fieldNumber: 1)
```

Comment:
This seems to be the cause of a small but definite reduction in generated code size.  I'm not sure if it's the simpler call sequence or not referring to the `FieldType` that does it.